src/forklift/__main__.py

```diff
@@ -59,19 +59,20 @@
                                                                             use the data that was updated rather than the entire ArcGIS Server instance. Note: this
                                                                             will only work for pallets that are in the warehouse.
     forklift speedtest                                                      Test the speed on a predefined pallet.
-'''
+"""
+
 #: check for Pro license
 try:
-    import arcpy
+    pass
```

Comment:
**question**: should this have been changed?

Reply:
Great catch! I'll fix this.